qrisk/stats.py

```diff
@@ -102,45 +107,21 @@ def cumulate_returns(x):
         return cum_returns(x)[-1]
 
     if convert_to == WEEKLY:
-        return df_daily_rets.groupby(
+        return returns.groupby(
```

Comment:
note: out of scope for this pr, but if this only supports pandas 18 we can replace these groupbys with resamples which will be much faster